library_generation/utilities.sh

```diff
@@ -174,17 +165,18 @@ download_from() {
   curl -LJ -o "${save_as}" --fail -m 30 --retry 2 "$url" || download_fail "${save_as}" "${repo}"
 }
 
+# copies the specified file in $1 to $2
+# will print "true" if $1 (local_repo) points to a non-existent file
 copy_from() {
   local local_repo=$1
   local save_as=$2
-  cp "${local_repo}" "${save_as}" || \
-    download_fail "${save_as}" "maven local"
+  cp "${local_repo}" "${save_as}" && echo "false" || echo "true"
```

Comment:
what does this line mean?

It seems to echo `true` if `cp` failed.

Reply:
I added a comment on top of the function. I will modify the comment for it to have a better explanation

Reply:
also added a variable to hold the result in order to improve readability